source/Engine.cpp

```diff
@@ -790,6 +790,20 @@ void Engine::Step(bool isActive)
 			info.SetCondition("flagship solar display");
 			info.SetString("flagship solar", to_string(flagshipSolar));
 		}
+		// Display combined Solar Power for system
+		int displaySystemPower = (flagship->DisplaySystemSolar() * 100); // Multiplied to better display decimal inputs
+		if(displaySystemPower >= 0.01 && flagship->Attributes().Get("solar scan"))
+		{
+			info.SetCondition("system solar display");
+			info.SetString("system solar", to_string(displaySystemPower));
+		}
+		// Display combined Solar Wind for system
+		int displaySystemWind = (flagship->DisplaySystemWind() * 100); // Multiplied to better display decimal inputs
+		if(displaySystemWind >= 0.01 && flagship->Attributes().Get("ramscoop scan"))
```

Comment:
I think it might be less confusing to call this "wind scan" or perhaps "solar wind scan"; although in following with my comment about the solar scan, maybe "wind sensor" or "solar wind sensor"?

Reply:
Sensor is fine, preferably "solar power sensor" and "solar wind sensor" so it's precise in what attribute each is sensing. Or if making them separate attributes is overkill then just "solar sensor"; I was just following the example of the tactical and strategic scanner modularity.

Reply:
I personally like having them as separate values. :) could be good to have some fraction that just tracks, say, solar power.